aptos-move/aptos-sdk-builder/src/rust.rs

```diff
@@ -874,46 +861,18 @@ fn decode_{}_argument(arg: TransactionArgument) -> Option<{}> {{
             U64 => "u64".into(),
             U128 => "u128".into(),
             Address => "AccountAddress".into(),
-            Vector(type_tag) => match type_tag.as_ref() {
-                U8 => {
-                    if local_types {
-                        "Vec<u8>".into()
-                    } else {
-                        "Bytes".into()
-                    }
```

Comment:
no concerns?

Reply:
exactly the same. Bytes is defined as a `Vec<u8> ` type in RUST.